kolena/_utils/dataframes/transformers.py

```diff
@@ -60,10 +63,88 @@ def _normalize_dicts(dict_list: List[Dict[str, Any]], sep: str = ".", max_level:
     return flattened_dicts
 
 
+def _try_parse(value: Any) -> Any:
```

Comment:
i think we need to use this function in `_deserialize_dataobject_str` or somewhere else

Reply:
Ya, this function should be used in https://github.com/kolenaIO/kolena/blob/trunk/kolena/io.py#L61